web/client-api/src/test/java/io/deephaven/web/client/api/PartitionedTableTestGwt.java

```diff
@@ -151,21 +148,19 @@ public void testTickingTransformedPartitionedTable() {
 
                     return partitionedTable.getKeyTable().then(keyTable -> {
                         keyTable.setViewport(0, 99, keyTable.getColumns(), null);
-                        return keyTable.getViewportData().then(data -> {
-                            assertEquals(0d, keyTable.getSize());
-
-                            return waitForEventWhere(keyTable, JsTable.EVENT_UPDATED,
-                                    (CustomEvent<ViewportData> d) -> d.detail.getRows().length == 5, 20004);
-                        }).then(event -> partitionedTable.getTable("2")).then(constituentTable -> {
-                            assertEquals(2, constituentTable.getColumns().length);
-                            assertEquals(2d, constituentTable.getSize());
-
-                            keyTable.close();
-                            constituentTable.close();
-                            partitionedTable.close();
-
-                            return null;
-                        });
+                        return keyTable.getViewportData()
+                                .then(data -> waitForEventWhere(keyTable, JsTable.EVENT_UPDATED,
```

Comment:
Yea, I'll drop it. I'm going to drop it from the Hierarchical test as well.
The problem with the getViewportData is that it could be getting the viewport data before the viewport has actually ticked in.
I noticed there was a `sleep` method in the HierarchicalTest script, but that seems to just make the script take longer to run - the table isn't actually ticking in that time. Unless I'm missing something there?

Reply:
Yeah the sleep was ostensibly to guarantee that at least the first row was there, but I agree we don't need it the way the test was updated in this PR.